src/System.IO.FileSystem/src/System/IO/Win32FileSystemEnumerable.cs

```diff
@@ -122,7 +123,7 @@ internal Win32FileSystemEnumerableIterator(string path, string originalUserPath,
             Debug.Assert(searchOption == SearchOption.AllDirectories || searchOption == SearchOption.TopDirectoryOnly);
             Debug.Assert(resultHandler != null);
 
-            _oldMode = Interop.Kernel32.SetErrorMode(Interop.Kernel32.SEM_FAILCRITICALERRORS);
+            _setBackOldMode = Interop.Kernel32.SetThreadErrorMode(Interop.Kernel32.SEM_FAILCRITICALERRORS, out _oldMode);
```

Comment:
Any of the IO ones, in theory. I'm trying to actually reproduce the prompting behavior and I'm not having much luck. On a thread with Windows folks to try and get the full skinny. I'll open an issue to track it- we'd probably want to refactor this a bit to wrap the P/Invokes.